src/Model/Contact.php

```diff
@@ -1662,6 +1662,28 @@ public static function getDefaultAvatar(array $contact, string $size)
 		return DI::baseUrl() . $default;
 	}
 
+	/**
+	 * Get avatar link for given contact id
+	 *
+	 * @param integer $cid contact id
+	 * @return string avatar link
+	 */
+	public static function getAvatarForContact(int $cid):string
```

Comment:
We're already in the `Contact` class so we don't need to repeat it in the method name, however the expected return and parameter can be mentioned.
```suggestion
	public static function getAvatarUrlForId(int $cid):string
```